how-to/workspace-platform-starter/client/src/framework/platform/interopbroker.ts

```diff
@@ -1227,26 +1562,42 @@ export function interopOverride(
 		 * Can we add an unregistered app.
 		 * @param clientIdentity The client identity.
 		 * @param intentName The intent name.
+		 * @param contextType The context type.
 		 * @returns True if we can add the app.
 		 */
 		private async canAddUnregisteredApp(
 			clientIdentity: OpenFin.ClientIdentity,
-			intentName?: string
+			intentName?: string,
+			contextType?: string
 		): Promise<boolean> {
+			let canAdd = false;
```

Comment:
Why do we need the canAdd var, can't we just return the values as before, otherwise I have to keep looking at the previous code to determine what it's going to return ?

Reply:
done